### ISSUE TITLE
Error in examples

### ISSUE THREAD

**Issue by baldodavi** [nextcloud/docker#1668]
Trying to use .examples to have a good installation of Nextcloud using docker-compose I found 2 bugs using both mariadb-fpm and mariadb-apache:
1- Letsencrypt supply only invalid certificate (self-signed)
2- Nginx always go in 500 internal error.

Here my compose code:
```
version: '3'

services:
  db:
    image: mariadb:10.5
    command: --transaction-isolation=READ-COMMITTED --binlog-format=ROW
    restart: always
    volumes:
      - db:/var/lib/mysql
    environment:
      - MYSQL_ROOT_PASSWORD=password.root
    env_file:
      - db.env

  redis:
    image: redis:alpine
    restart: always

  app:
    image: nextcloud:apache
    restart: always
    volumes:
      - nextcloud:/var/www/html
    environment:
      - VIRTUAL_HOST=cloud.mydomain.com
      - LETSENCRYPT_HOST=cloud.mydomain.com
      - LETSENCRYPT_EMAIL=mymail@gmail.com
      - MYSQL_HOST=db
      - REDIS_HOST=redis
    env_file:
      - db.env
    depends_on:
      - db
      - redis
    networks:
      - proxy-tier
      - default

  cron:
    image: nextcloud:apache
    restart: always
    volumes:
      - nextcloud:/var/www/html
    entrypoint: /cron.sh
    depends_on:
      - db
      - redis

  proxy:
    build: ./proxy
    restart: always
    ports:
      - 80:80
      - 443:443
    volumes:
      - certs:/etc/nginx/certs:ro
      - vhost.d:/etc/nginx/vhost.d
      - html:/usr/share/nginx/html
      - /var/run/docker.sock:/tmp/docker.sock:ro
    networks:
      - proxy-tier

  letsencrypt-companion:
    image: nginxproxy/acme-companion
    restart: always
    volumes_from:
      - proxy
    volumes:
      - certs:/etc/nginx/certs:rw
      - acme:/etc/acme.sh
      - /var/run/docker.sock:/var/run/docker.sock:ro
    networks:
      - proxy-tier
    depends_on:
      - proxy

# self signed
#  omgwtfssl:
#    image: paulczar/omgwtfssl
#    restart: "no"
#    volumes:
#      - certs:/certs
#    environment:
#      - SSL_SUBJECT=servhostname.local
#      - CA_SUBJECT=my@example.com
#      - SSL_KEY=/certs/servhostname.local.key
#      - SSL_CSR=/certs/servhostname.local.csr
#      - SSL_CERT=/certs/servhostname.local.crt
#    networks:
#      - proxy-tier

volumes:
  db:
  nextcloud:
  certs:
  acme:
  vhost.d:
  html:

networks:
  proxy-tier:

```

Obliviously mydomain and mymail are correct and I've fill password on db.env.

Thanks.
David.

**Comment by baldodavi** [comment]
Seems cause is a heavy rate with letsencrypt. Is there a way to have staging?

**Comment by sparkitny** [comment]
Hi,

yesterday I updated my installation. Unfortunately, it still doesn't work. But I went through several errors including the `volumes_from` option. In compose files starting from version 3 this option is not available anymore. So the above mentioned compose file cannot work.

**Comment by Sputnik2525** [comment]
Hi,

I have the same error, volumes_from results in an error when building the docker-compose.
`ERROR: The Compose file './docker-compose.yml' is invalid because:
Unsupported config option for services.letsencrypt-companion: 'volumes_from'`

Link to [documentation](https://docs.docker.com/compose/compose-file/compose-file-v3/#volumes)

**Comment by Lejo1** [comment]
About volumes_from:
Sorry, I somehow only tested the change for version 2. This PR reverts the change: #1677

**Comment by joshtrichards** [comment]
> Seems cause is a heavy rate with letsencrypt. Is there a way to have staging?

The LE-acme-companion image isn't ours. You'd have to refer to their documentation. It looks like there is a `LETSENCRYPT_TEST` option. Perhaps that'll do it?

https://github.com/nginx-proxy/acme-companion/blob/main/docs/Let's-Encrypt-and-ACME.md#acme-accounts-handling

The `volumes_from` stuff was addressed in #1677.

Closing since there's nothing actionable for us to do here.
